charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     - 'secret'
     - 'downwardAPI'
     - 'persistentVolumeClaim'
-  hostNetwork: false
+  hostNetwork: {{ include "newrelic.common.hostNetwork.value" . }}
```

Comment:
Hhhmm... I did not think about the PSP. I would revisit them on the rest of the charts.

Good catch.